serialization.md

```diff
@@ -422,14 +362,13 @@ bin/my-app-binary
 bin/my-app-tools
 ```
 
-The TarSum checksum for the archive file is then computed and placed in the
-JSON metadata along with the execution parameters.
+The TarSum checksum for the archive file is then computed and placed in the JSON metadata along with the execution parameters.
```

Comment:
Oh sure I can. This `serialization.md` needs loads more. It seemed like we may punt, to make headway on `manifest.md` first.
I'll do a once-over of this doc right now.